include/boost/json/conversion.hpp

```diff
@@ -322,6 +322,9 @@ struct is_described_class;
 template<class T>
 struct is_described_enum;
```

Comment:
is this header included in `<boost/json.hpp>`?

Reply:
Yes